src/System.Security.Cryptography.Pkcs/ref/System.Security.Cryptography.Pkcs.netcoreapp.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System.Security.Cryptography.Pkcs
+{
+    public sealed partial class SignerInfo
```

Comment:
Is this the only net new API? The other new APIs already exist on .NET Framework?

Reply:
Yeah, everything else has been in .NET Framework for a while; these two members on SignerInfo were reviewed a couple months ago, and are new to netcoreapp.